### PR TITLE
Add support to SR Linux IXR-H2/H3 devices, resolves #623

### LINKED ISSUE #623: Add IXR-H2 and IXR-H3 SR Linux models
Need to add newer Chassis variants for SR Linux devices

### PR DESCRIPTION
**Pull request by sacckth**
Work in progress...
Add support to SR Linux IXR H3 node

### CODE REVIEW

**Comment by sacckth** [comment]
Resolves #623

File: docs/manual/kinds/srl.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ There are many ways to manage SR Linux nodes, ranging from classic CLI managemen
 ### Types
 For SR Linux nodes [`type`](../nodes.md#type) defines the hardware variant that this node will emulate.
 
-The available type values are: `ixr6`, `ixr10`, `ixrd1`, `ixrd2`, `ixrd3` which correspond to a hardware variant of Nokia 7250/7220 IXR chassis.
+The available type values are: `ixr6`, `ixr10`, `ixrd1`, `ixrd2`, `ixrd3`, `ixrh3` and `ixrh2` which correspond to a hardware variant of Nokia 7250/7220 IXR chassis.
```
Comment (hellt) [inline docs/manual/kinds/srl.md:39]: Hi Sergio, can you make h2 before h3 to follow the progression?

File: nodes/srl/srl.go [hunk 2]
```diff
@@ -66,6 +66,8 @@ var (
 		"ixrd1": "7220IXRD1.yml",
 		"ixrd2": "7220IXRD2.yml",
 		"ixrd3": "7220IXRD3.yml",
```
Comment (hellt) [inline nodes/srl/srl.go:68]: H2 before H3

File: schemas/clab.schema.json [hunk 3]
```diff
@@ -193,7 +193,9 @@
                             "ixr10",
                             "ixrd1",
                             "ixrd2",
-                            "ixrd3"
+                            "ixrd3",
+                            "ixrh3",
```
Comment (hellt) [inline schemas/clab.schema.json:197]: H2 before H3

**Comment by hellt** [comment]
Thanks @sacckth !

**Pull request merged**